components/ui/dropdown.vue

```diff
@@ -1,16 +1,33 @@
 <template>
-  <div :class="isOpen && 'open'" class="dropdown-shell">
+  <div :class="{open: isOpen}" class="dropdown-shell">
```

Comment:
Let's wrap classes in single quotes. On the line 14 you put it, so do it everywhere for the sake of code consistency

Reply:
The point is that it is a key in an object, which means that it needs to be wrapped in quotes when it contains any special characters (like 'right-edge' in line 14) but not otherwise